### PR TITLE
Set FBA, NeoGeo and Arcade to use MAME Scraper

### PR DESCRIPTION
**Pull request by zerojay**
Set up scraper to use the -mame parameter (scrape by filename) when scraping system names beginning with "fba", "neogeo" and "arcade" as mentioned on https://github.com/sselph/scraper.

### CODE REVIEW

File: scriptmodules/supplementary/scraper.sh [hunk 1]
```diff
@@ -47,7 +47,7 @@ function scrape_scraper() {
     params+=(-output_file "$gamelist")
     params+=(-rom_dir "$romdir/$system")
     params+=(-workers "4")
-    [[ "$system" =~ ^mame- ]] && params+=(-mame -mame_img t,m,s)
+    [[ "$system" =~ ^mame- ]] || [[ "$system" =~ ^fba ]] || [[ "$system" =~ ^neogeo ]] || "$system" =~ ^arcade ]] && params+=(-mame -mame_img t,m,s)
```
Comment (joolswills) [inline scriptmodules/supplementary/scraper.sh]: We only use ^mame- to match all the rom folders prefixed with mame - if you want to use others just either match individual strings or use a regular expression with |

`[[ "$system" =~ ^mame- || "$system" =~ fba|neogeo|arcade ]]`

**Comment by joolswills** [comment]
I don't think we have an "arcade" system either currently either.

File: scriptmodules/supplementary/scraper.sh (hunk 1 above)
Comment (joolswills) [inline scriptmodules/supplementary/scraper.sh]: can probably be simplified to

`[[ "$system" =~ ^mame-|fba|neogeo ]]`

**Comment by zerojay** [comment]
Indeed, there is not an "arcade" system, however one of the popular multiboot SD card distribuions has custom made an arcade system which allows users to consolidate all their MAME/FBA/NeoGeo games into one set. As it's such a simple change in RP, I figure it wouldn't be a problem to add it to the list of possible supported systems for the scraper, but I'll let you make a decision on that.

File: scriptmodules/supplementary/scraper.sh (hunk 1 above)
Comment (zerojay) [inline scriptmodules/supplementary/scraper.sh]: Apologies, forgot that we now just have a single fba rom directory.

**Comment by joolswills** [comment]
I will leave out arcade for now - if we ever add it ourselves we can put it in. Please can you squash these commits and I will merge. thanks.

**Comment by zerojay** [comment]
Closing as requested.

**Comment by joolswills** [comment]
I'm not asking for it to be closed. I wanted the commits squashed and force pushed back to the branch. I'll manually implement the changes though. thanks.

**Comment by joolswills** [comment]
I squashed your commits and merged. thanks.

**Pull request closed without merging**